include/flatbuffers/flatbuffers.h

```diff
@@ -1967,6 +1967,9 @@ class FlatBufferBuilder {
 
   /// @cond FLATBUFFERS_INTERNAL
   template<typename T> struct StructKeyComparator {
+    StructKeyComparator() {}
+    StructKeyComparator(const StructKeyComparator &) {}
```

Comment:
It looks unusual, enabled copy constructor with the disabled assignment operator. It might be better to delete the assignment operator. It will unblock all implicitly generated constructors and assignment operators.
@aardappel is there any reason to disable the copy assignment operator?
This object is used with `std::sort` only.

Reply:
This was added by @tianyapiaozi .. I am not sure what the reason was. To me simplest would be to go with all default/implicit constructors and remove `FLATBUFFERS_DELETE_FUNC` below, but I am fine with whatever solution works.

Reply:
I have removed the operator=() deletion instead.